build.gradle.kts

```diff
@@ -92,6 +92,10 @@ sonar {
                 "dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrDataModel.ts," +
```

Comment:
General comment: None of the percentage fields in the DataModel end with percentage. I thought this was our new policy?

Reply:
Took this comment to the other PR.

Reply:
Will be changed in the PR of DALA_2307

Reply:
This is done